### PR TITLE
Update homepage to match latest copy

### PR DESCRIPTION
**Pull request by solazio**
## Done

Update homepage to match latest copy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2674

## Screenshots

image]

image

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -27,6 +30,257 @@ <h1>
...
+          <p class="p-stepped-list__content u-stepped-list-margin">
+            If you mainly use MicroK8s you can make our kubectl the default one on your command-line with <code>alias mkctl=”microk8s kubectl”</code>. Since it is a standard upstream kubectl you can also drive other Kubernetes clusters with it by pointing to the respective kubeconfig file via the <code>--kubeconfig</code> argument.
+          </p>
+        </li>
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Access the Kubernetes dashboard</h3>
+
+          <p class="p-stepped-list__content u-stepped-list-margin">
+            <a href="https://ubuntu.com/tutorials/install-a-local-kubernetes-with-microk8s#4-accessing-the-kubernetes-and-grafana-dashboards" class="p-link--external">Follow the detailed tutorial&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:110]: Can you remove `&nbsp;&rsaquo;` please

File: templates/index.html [hunk 2]
```diff
@@ -27,6 +30,257 @@ <h1>
...
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Turn on the services you probably want</h3>
+
+          <div class="p-code-copyable u-stepped-list-margin">
+            <input class="p-code-copyable__input" aria-label="microk8s enable dashboard registry dns istio" value="microk8s enable dashboard registry dns istio" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p class="p-stepped-list__content u-stepped-list-margin">
+            Similarly, <code>microk8s disable</code> turns off a service. Try <code>microk8s enable --help</code> for a list of available services and optional features.
```
Comment (anthonydillon) [inline templates/index.html:175]: s/and optional features/built in/

File: templates/index.html [hunk 3]
```diff
@@ -27,6 +30,257 @@ <h1>
...
+            <input class="p-code-copyable__input" aria-label="microk8s kubectl get all --all-namespaces" value="microk8s kubectl get all --all-namespaces" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+        </li>
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Access the Kubernetes dashboard</h3>
+
+          <p class="p-stepped-list__content u-stepped-list-margin">
+            <a href="https://ubuntu.com/tutorials/install-a-local-kubernetes-with-microk8s#4-accessing-the-kubernetes-and-grafana-dashboards" class="p-link--external">Follow the detailed tutorial&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:192]: Can you remove `&nbsp;&rsaquo;` please

File: templates/index.html [hunk 4]
```diff
@@ -27,6 +30,257 @@ <h1>
...
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Turn on the services you probably want</h3>
+
+          <div class="p-code-copyable u-stepped-list-margin">
+            <input class="p-code-copyable__input" aria-label="microk8s enable dashboard registry dns istio" value="microk8s enable dashboard registry dns istio" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p class="p-stepped-list__content u-stepped-list-margin">
+            Similarly, <code>microk8s disable</code> turns off a service. Try <code>microk8s enable --help</code> for a list of available services and optional features.
```
Comment (anthonydillon) [inline templates/index.html:250]: s/and optional features/built in

File: templates/index.html [hunk 5]
```diff
@@ -27,6 +30,257 @@ <h1>
...
+            <input class="p-code-copyable__input" aria-label="microk8s kubectl get all --all-namespaces" value="microk8s kubectl get all --all-namespaces" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+        </li>
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">Access the Kubernetes dashboard</h3>
+
+          <p class="p-stepped-list__content u-stepped-list-margin">
+            <a href="https://ubuntu.com/tutorials/install-a-local-kubernetes-with-microk8s#4-accessing-the-kubernetes-and-grafana-dashboards" class="p-link--external">Follow the detailed tutorial&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:267]: Please remove `&nbsp;&rsaquo;`

File: templates/index.html [hunk 6]
```diff
@@ -5,20 +5,23 @@
...
       </h1>
 
       <p class="p-heading--four u-sv3">
-        Super-simple production-grade upstream K8s. One-command install on 42 flavours of Linux. Made for developers and devops. Great for edge and IoT.
+        The simplest production-grade upstream K8s. Single command install on Linux, Windows and macOS. Made for devops, great for edge, appliances and IoT.
       </p>
 
-      <p><a class="p-button--positive is-inline u-no-margin--left" href="#get-started">Get started</a>&nbsp;&nbsp;for Linux, Windows or macOS</p>
+      <p>
+        <a class="twitter-share-button" href="https://twitter.com/intent/tweet" data-size="large" data-hashtags="microk8s">Tweet</a>
```
[the remark this answers is not in the document]
Comment (solazio) [reply templates/index.html:16]: the one that I used is the [official Twitter 'tweet'](https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview) button. The page we link to in vanilla doesn't exist.

File: templates/index.html [hunk 7]
```diff
@@ -5,20 +5,24 @@
...
+      </p>
 
       <div>
         <a class="github-button" data-size="large" href="https://github.com/ubuntu/microk8s" aria-label="Follow @ubuntu on GitHub"></a>
         <a class="github-button" data-size="large" href="https://github.com/ubuntu/microk8s" data-icon="octicon-star" data-show-count="true" aria-label="Star ubuntu/microk8s on GitHub"></a>
         <a class="github-button" data-size="large" href="https://github.com/ubuntu/microk8s/fork" data-icon="octicon-repo-forked" data-show-count="true" aria-label="Fork ubuntu/microk8s on GitHub"></a>
+        <a class="twitter-share-button" href="https://twitter.com/intent/tweet" data-size="large" data-hashtags="microk8s">Tweet</a>
```
Comment (anthonydillon) [inline templates/index.html:23]: There are two now.

**Comment by spencerbygraves** [comment]
@solazio Do the nav items have the right padding? I thought it was 1rem left and right, could you check please?

For the Twitter button, the guidance currently in Vanilla for sharing is to use an official asset. Here's what they look like in the header based on the doc feedback:

image

Or we can use the Github buttons without numbers to save a bit of space:

image

Do you have a preference @pmahnke?

**Comment by pmahnke** [comment]
@spencerbygraves I think the numbers are important.  And doesn't really take up a LOT more space.

**Comment by solazio** [comment]
@spencerbygraves would you be able to create a mobile design please?

**Pull request merged**